db/migrate/20190820095253_copy_enrichments_to_next_cycle.rb

```diff
@@ -0,0 +1,20 @@
+class CopyEnrichmentsToNextCycle < ActiveRecord::Migration[5.2]
+  def up
+    current_providers = RecruitmentCycle.current_recruitment_cycle.providers.includes(:enrichments).all
+    next_providers = RecruitmentCycle.next_recruitment_cycle.providers.all
+
+    current_providers.each do |p|
+      next if p.enrichments.empty?
+
+      last_enrichment = p.enrichments.order(created_at: :desc, id: :desc).first.dup
+
+      next_cycle_provider = next_providers.find { |n| n.provider_code == p.provider_code }
+      next_cycle_provider.enrichments << [last_enrichment]
```

Comment:
before assigning

`last_published_at` needs to be nil out
`status` must not be `published`